app/renderer/polyfills/index.js

```diff
@@ -0,0 +1,35 @@
+let log, settings, clipboard, shell, remote, ipcRenderer,
+    i18NextBackend, i18NextBackendOptions;
+
+function buildForBrowser () {
+  if (process.env.BUILD_BROWSER) {
```

Comment:
we could reduce this whole function body to a single return

Reply:
i know, i found this a bit easier to follow